src/routes/dispensary/add/+page.server.ts

```diff
@@ -1,7 +1,39 @@
+import supabase from "$lib/supabase/client";
+
 export const actions = {
   create: async ({ request }) => {
-    const data = await request.formData()
-    const formName = data.get("name")
-    return { success: true, name: formName }
-  }
-}
+    try {
+      const data = await request.formData();
+      const formName = data.get("name");
+      const formAddress = data.get("address");
+      const formCity = data.get("city");
+      const formState = data.get("state");
+      const formZip = data.get("zip");
+      const formStatus = data.get("status");
```

Comment:
I don't see the purpose of letting the user fill out `status` and `createdBy` in the form.  Instead, we should just pass fixed values below, like
```
 .insert([
          {
            name: formName,
            address_1: formAddress,
            city: formCity,
            region_id: formState,
            postal_code: formZip,
            validation_status_id: 2, // Pending
            created_by: '22222222-2222-2222-2222-222222222222', // placeholder user id
          },
        ]);
```